dotcom-rendering/src/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -270,41 +277,45 @@ export const canShowRRBanner: CanShowFunctionType<BannerProps> = async ({
 		return { show: false };
 	}
 
-	const { module, meta } = response.data;
+	const { module } = response.data;
+
+	const { props, name } = module;
 
 	const fetchEmail = isSignedIn
 		? lazyFetchEmailWithTimeout(idApiUrl)
 		: undefined;
 
-	return { show: true, meta: { module, meta, fetchEmail } };
-};
-
-export type BannerProps = {
-	meta: TestTracking;
-	module: ModuleData;
-
-	fetchEmail?: () => Promise<string | null>;
-};
-
-type RemoteBannerProps = BannerProps & {
-	componentTypeName: ReaderRevenueComponentType;
-	displayEvent: string;
+	return {
+		show: true,
+		meta: {
+			props: {
```

Comment:
here the client adds some extra fields to the props. Previously this was done further down in `ReaderRevenueBanner`